test/integration/addons_test.go

```diff
@@ -31,7 +31,8 @@ import (
 
 var (
 	addonManagerCmd = []string{"get", "pod", "kube-addon-manager-127.0.0.1", "--namespace=kube-system"}
-	dashboardCmd    = []string{"get", "rc", "kubernetes-dashboard", "--namespace=kube-system"}
+	dashboardRcCmd  = []string{"get", "rc", "kubernetes-dashboard", "--namespace=kube-system"}
+	dashboardSvcCmd = []string{"describe", "svc", "kubernetes-dashboard", "--namespace=kube-system"}
```

Comment:
I pulled in your change to test it out, I think you need to change this to "get", "svc". We parse the output using -o=json, and describe doesn't support that flag.